geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/data/RemoveSchemaCommand.scala

```diff
@@ -46,7 +46,7 @@ trait RemoveSchemaCommand[DS <: GeoMesaDataStore[_, _, _ ,_]] extends DataStoreC
         } else {
           println(s"Removing '$typeName'")
           ds.removeSchema(typeName)
-          if (ds.getSchema(typeName) != null) {
+          if (ds.getTypeNames.contains(typeName)) {  // NB: For Cassandra this only checks metadata
```

Comment:
does `getSchema` not return null for the cassandra data store? I'd like to keep it as `getSchema` vs `getTypeNames` b/c for accumulo and hbase `getTypeNames` scans the whole metadata table instead of a single row.